migrate/labels.yml

```diff
@@ -0,0 +1,118 @@
+# Repository labels to be synced across the entire org.
+# Every item must have a name, color, and description.
+
+# Notes:
+# * Repositories may define their own labels as long as they don't
+#   conflict with what's listed here. If it turns out a repository-specific
+#   label would be useful to have in the entire organization, we can always add
+#   it here post facto.
+# * We have added these based on need. They are not all-encompassing. If something
+#   seems missing, feel free to open a PR.
+# * Whenever a repo has a matching label, that label will be updated. If there
+#   is no matching label, the label will be created anew. "Matching" is determined
+#   by comparing the label's lowercased name, with all puncutation, emoji, and special
+#   chars removed.
+# * For picking hex colors, you might use: https://www.htmlcolor-picker.com/
```

Comment:
@kdmccormick It might be a temporary issue but this site was unreachable for me when I tried to access it today.

If necessary, https://www.color-hex.com/ might be a good alternative to mention here.

Reply:
Good call, I like that site better anyway, thanks!